### DISCUSSION TITLE
Designated instructions for publishing working samples - with json, screenshots, and blurb

### DISCUSSION THREAD

**Discussion by aaronsteers** [langflow-ai/langflow#200] [Ideas]
I'm just kicking the tires here and I would love to have a library of samples - or really any samples at all to help me understand the workflows.

1. Are there any existing samples, either in the repo or in another location?
2. Do I understand correctly that a sample working case could be exported as json and then shared via Gist or another mechanism?
3. Assuming on the right track as above: what if we create a discussion category for users to share working samples? Specifically I'm thinking these three components: (1) A link to the Gist or repo URL that contains the json, (2) a high res screenshot of the LangFlow canvas, (3) a few short sentences or bulletpoints about what the sample does.

What do folks think?

**Comment by GeeksikhSecurity** [comment]
I came across Flowwise which is similar and looks like made by the same developers. The prebuilt templates in their Marketplace give a starting point. See https://flowiseai.com/ and https://www.youtube.com/watch?v=eKb0A9vHy7E

**Reply by GeeksikhSecurity** [reply]
Found a good thread on challenges with Langchain and Langflow, which explains by the developers created a new tool flowwise.
Another tool ChainFury is an open-source project, and is currently in the alpha stage. 
The first and foremost was that langchain is very hard to use:

    there is no straight forward syntax
    interactivity with chats is not built in
    support for multiple modalities is hard
https://github.com/NimbleBoxAI/ChainFury

![image](https://user-images.githubusercontent.com/37025034/236653248-486b4f8d-3b4b-4fe3-836b-9fc0d599430e.png)

**Comment by ogabrielluiz** [comment]
Hi! Sorry for the delayed response.

https://github.com/logspace-ai/langflow_examples

What do you think of this approach? We've released this a few days ago.

**Reply by aaronsteers** [reply]
Very cool. Thanks for creating and sharing this, @ogabrielluiz . 👍 

Yes, I think this is a big help!
